Update mixer docs (#2738)
pygame-community/pygame-ce: docs/reST/ref/mixer.rst

   | :sg:`Channel(id) -> Channel`

   Return a Channel object for one of the current channels. The id must be a

>    value from 0 to the value of ``pygame.mixer.get_num_channels()`` (exclusive).

Reviewer: ```suggestion
   value from 0 upto but not including ``pygame.mixer.get_num_channels()``.
```
The word "exclusive" isn't very clear, and may mislead people to think that both ends aren't included

Reviewer 2: Good idea.

But I'd say it as "up to" rather than "upto"

>    value from 0 to the value of ``pygame.mixer.get_num_channels()`` (exclusive).

Reviewer 3: ```suggestion
   value from 0 up to, but not including, ``pygame.mixer.get_num_channels()``.
```
